mgmt/rest/rest_func_test.go

```diff
@@ -584,8 +585,10 @@ func TestPluginRestCalls(t *testing.T) {
 				So(err, ShouldBeNil)
 				if len(SNAP_AUTODISCOVER_PATH) == 0 {
 					if len(SNAP_PATH) != 0 {
-						SNAP_AUTODISCOVER_PATH = fmt.Sprintf("%s/plugin", SNAP_PATH)
-						log.Warning(fmt.Sprintf("SNAP_AUTODISCOVER_PATH has been set to SNAP_PATH/plugin (%s). This might cause test failures", SNAP_AUTODISCOVER_PATH))
+
+						SNAP_PATH = helper.BuildPath
```

Comment:
This seems redundant since SNAP_PATH is set to helper.BuildPath at the top of the file.